liferay-workspace/modules/blade.indexerpostprocessor/src/main/resources/META-INF/module-log4j.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
We haven't added logging for any of the samples yet.  We should likely do this for all of them, but probably in a separate pull request.